lua/terrortown/lang/en.lua

```diff
@@ -2222,6 +2222,11 @@ L.help_locational_voice_prep = [[By default the proximity chat is disabled in th
 Note: Proximity chat is always disabled during the post round phase.]]
 L.help_voice_duck_spectator = "Ducking spectators makes other spectators quieter in comparison to living players. This can be useful if one wants to listen closely to the discussions of the living players."
 
+L.help_equipmenteditor_clip_on_buy = [[The clip size on buy defines the amount of uses the weapon has when bought in the shop.
+
+Note: This setting is only available for weapons that enable this feature.]]
```

Comment:
If Initialize is not overriden it also works, right?

Reply:
If it is overridden it breaks, yes. But it also has to be enabled on a per weapon basis because most weapons would "break" if the clip is configurable